src/streamlit_app.py

```diff
@@ -535,6 +536,25 @@ def find_item(
                     return item
         return None
 
+    def get_order_item_caption(
+        json_data: typing.Any,
+        name: str,
+        order: str,
+    ) -> typing.Any:
+        parts = [" * "]
+
+        name = name.strip('"')
+        order = "ascending" if order == "asc" else "descending"
+
+        item = find_item(json_data, name)
+
+        if item is not None and item["ui_url"] is not None and item["ui_url"] != "":
```

Comment:
`item.get("ui_url")` instead of `item["ui_url"]` to avoid keyerror

Reply:
Fixed